controllers/controller.games.js

```diff
@@ -1,12 +1,24 @@
 const categories = require('../db/data/test-data/categories');
-const { fetchCategories } = require('../models/model.games.js');
+const { fetchCategories, fetchReviewById } = require('../models/model.games.js');
 
 exports.getCategories = (req, res, next) => {
   fetchCategories().then((categories) => {
     res.status(200).send({ categories });
   });
 };
 
+exports.getReviewById = (req, res, next) => {
+  const { review_id } = req.params;
+
+  fetchReviewById(review_id)
+    .then((review) => {
+      res.status(200).send({ review });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
 exports.noPath = (req, res) => {
```

Comment:
If we are going to export this as a controller, I think it would make more sense for it to be in an errors folder, or just in app.js until you make one

Reply:
done (moved to error handling file)